projects/storefrontstyles/scss/components/cart/_applied-coupons.scss

```diff
@@ -47,14 +47,16 @@
 
   .cx-coupon-apply {
     display: inline-flex;
-    align-items: center;

```

Comment:
keep this to center center the container, because right now it's not vertical-aligned to the center